package.json

```diff
@@ -30,7 +30,7 @@
     "node": ">=4.0.0"
   },
   "dependencies": {
-    "@justeat/fozzie-colour-palette": "^0.8.0",
+    "@justeat/fozzie-colour-palette": "^0.9.0",
```

Comment:
This version of `fozzie-colour-palette` needs to be published now that it has been merged. The Travis build is failing because the version in the Yarn lockfile does not match.